python/setup.py

```diff
@@ -1,27 +1,10 @@
 # Copyright (c) 2021-2023, NVIDIA CORPORATION. All rights reserved.
 # See file LICENSE for terms.
 
-import versioneer
 from setuptools import find_packages
 from skbuild import setup
 
 setup(
-    name="kvikio",
-    version=versioneer.get_version(),
-    description="KvikIO - GPUDirect Storage",
-    url="https://github.com/rapidsai/kvikio",
-    author="NVIDIA Corporation",
-    license="Apache 2.0",
-    classifiers=[
-        "Intended Audience :: Developers",
-        "Topic :: Database",
-        "Topic :: Scientific/Engineering",
-        "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 3.9",
-    ],
-    # Include the separately-compiled shared library
-    extras_require={"test": ["pytest", "pytest-xdist"]},
     packages=find_packages(exclude=["tests*"]),
```

Comment:
I would reuse the `find_packages` result for `package_data` to guarantee they stay in sync.